go/vm/geth/geth.go

```diff
@@ -101,9 +101,18 @@ func createGethInterpreterContext(parameters vm.Parameters) (*geth.EVM, *geth.Co
 	// Set hard forks for chainconfig
 	chainConfig := *params.AllEthashProtocolChanges
 	chainConfig.ChainID = new(big.Int).SetBytes(context.ChainID[:])
-	chainConfig.IstanbulBlock = big.NewInt(int64(vm.R07_Istanbul) * 10)
-	chainConfig.BerlinBlock = big.NewInt(int64(vm.R09_Berlin) * 10)
-	chainConfig.LondonBlock = big.NewInt(int64(vm.R10_London) * 10)
+	chainConfig.IstanbulBlock = big.NewInt(int64(vm.R07_Istanbul) * 1000)
```

Comment:
Can you update this to use [`GetForkBlock`](https://github.com/Fantom-foundation/Tosca/blob/84cbb23429c9275a1d890a1e3cd8615d2f448cdc/go/ct/common/revisions.go#L105) instead of hard-coded constants?

Reply:
This was a little too much work and is is still not 100% how I would like it.
A clean implementation should get rid first of one the vm.Revision and ct.Revision types

The proposed solution factors this code out and reuses an already existing function that was meat to do this. (although it was never used before) 
Please let me know if we should keep these changes or we revert to the hardcoded values.